rockr/views.py

```diff
@@ -12,7 +13,7 @@
     UserGoal,
     UserMusicalInterest,
 )
-
+from rockr.models.band import Band, UserBand
```

Comment:
you can add these to this line: `from rockr.models import User, auth0`. See `models/__init__.py`

Reply:
Added